src/settings.js

```diff
@@ -1350,6 +1350,10 @@ var EMIT_EMSCRIPTEN_METADATA = 0;
 // for effective code size optimizations to take place.
 var SUPPORT_ERRNO = 1;
 
+// If set to 1, a C++ exception that is thrown in Emscripten runtime will abort JS
+// animation callbacks from firing. If set to 0, thrown exceptions will not affect
+// animation callbacks.
```

Comment:
How about `If set to 1, if the program halts (due to a C++ exception, exit(), abort(), etc.) then asynchronous operations scheduled by the emscripten runtime will not be called. This is normally what you expect, if you split up your code into asynchronous pieces and one of them errors then it can be confusing to have the rest try to run later. Disabling this can be useful for code size, however.`